tst/EnergyPlus/api/TestDataTransfer.py

```diff
@@ -70,6 +70,7 @@ def time_step_handler(state):
             # val = api.exchange.list_available_api_data_csv()
             # with open('/tmp/data.csv', 'w') as f:
             #     f.write(val.decode(encoding='utf-8'))
+            data = api.exchange.get_api_data(state)
```

Comment:
And here's the super nice Python usage...one function, no weirdness, just what you would expect.